sound/soc/codecs/rt5682-i2c.c

```diff
@@ -273,6 +273,8 @@ static void rt5682_i2c_shutdown(struct i2c_client *client)
 {
 	struct rt5682_priv *rt5682 = i2c_get_clientdata(client);
 
+	cancel_delayed_work_sync(&rt5682->jack_detect_work);
```

Comment:
do we need this for the SoundWire case as well? we don't have a .shutdown() callback at the moment.

Reply:
Fixes tag should probably be 

````
Fixes: a50067d4f3c1d ('ASoC: rt5682: split i2c driver into separate module')
````

It'll be hard to port this to earlier versions since the code was split in two.

Reply:
> do we need this for the SoundWire case as well? we don't have a .shutdown() callback at the moment.

I checked rt5682-sdw.c, it is doing 'cancelling of delayed work' in remove function. So, seems ok for sdw driver.

Reply:
> Fixes tag should probably be
> 
> ```
> Fixes: a50067d4f3c1d ('ASoC: rt5682: split i2c driver into separate module')
> ```
> 
> It'll be hard to port this to earlier versions since the code was split in two.

Thank you for giving the full tag.  Added to description now.